src/helpers/permit.ts

```diff
@@ -0,0 +1,62 @@
+import { MaxUint256, PermitTransferFrom, SignatureTransfer } from "@uniswap/permit2-sdk";
+import { randomBytes } from "crypto";
+import { BigNumber, ethers } from "ethers";
+import { getBotConfig, getBotContext } from "../bindings";
+
+const PERMIT2_ADDRESS = "0x000000000022D473030F116dDEE9F6B43aC78BA3"; // same on all chains
+
+/**
+ * Generates permit2 signature data with `spender` and `amountInETH`
+ *
+ * @param spender The recipient address we're going to send tokens
+ * @param amountInETH The token amount in ETH
+ *
+ * @returns Permit2 url including base64 encocded data
+ */
+export const generatePermit2Signature = async (spender: string, amountInETH: string): Promise<string> => {
```

Comment:
```suggestion
export const generatePermit2Signature = async (spender: string, amountInEth: string): Promise<string> => {
```

Reply:
https://github.com/ubiquity/bounty-bot/pull/138/commits/ca57bf0d3032801ad10d854af0a2708080254b8f

`BigNumber` would be replaced with `BigInt` since the latest ethers version so I feel like `string` would be more extensible

Reply:
I'm pretty sure that the latest Ethers uses the built in JavaScript BigNumber primitive.

Reply:
Even with BigNumber, its a string internally. Lets use string instead of bignumber here.